### ISSUE TITLE
Find by aria-label

### ISSUE THREAD

**Issue by jaimeiniesta** [germsvel/phoenix_test#243]
It would be great to be able to find elements by its `aria-label` property, for example, this input should be findable by its `aria-label` "Your Name"

```html
<input id="name" type="text" name="name" aria-label="Your Name"/>
```

I think that's not possible currently. Are there plans to support ARIA attributes?

**Comment by germsvel** [comment]
Only part of this was done in #319, so I'm reopening this so we can close it once we fully support `aria-label` with all helper functions.
